### ISSUE TITLE
hyperspec miscompiles expressions like `DIV()`

### ISSUE THREAD

**Issue by catmando** [hyperstack-org/hyperstack#50]
For example

```ruby
mount 'Foo' do
  class Foo 
    include Hyperstack::Component
    render do
      DIV()
    end
  end
end
```

The expression `DIV()` gets "decompiled" to simply `DIV` (constant not method call).

This must be a problem in the parser_gem.  https://github.com/whitequark/parser/issues/529

**Comment by iliabylich** [comment]
@catmando May I ask why do you think that this code gets parsed as a constant? How did you identify that? Do you get something like a `NameError: uninitialized constant DIV`?

**Comment by iliabylich** [comment]
Also maybe I'm wrong but you use `unparser` in a few places, right? (I'm not familiar with your codebase, so I can be completely wrong) Maybe that's an issue with un-parsing? `DIV()` gets parser as a method call, but unparser converts it to `DIV` (because parentheses in ruby are optional). And after that it gets parsed back to the constant (because there are no parentheses anymore)

**Comment by catmando** [comment]
actually its in the unparser gem: https://github.com/mbj/unparser/issues/83

**Comment by catmando** [comment]
should be fixed in unparser, but for now there is a patch in hyperspec.

closed in 1.0.alpha1
